data-plane/src/stats_client.rs

```diff
@@ -55,6 +76,7 @@ impl StatsClient {
         let mem_info = sys_info::mem_info()?;
         let cpu = sys_info::loadavg()?;
         let cpu_num = sys_info::cpu_num()?;
+        let (allocated, free, max) = Self::get_file_descriptor_info()?;
```

Comment:
it feels slightly odd to fail all reporting if one of these fails, should we split this out to just log the errors that happen, and publish all metrics available?

Reply:
yeah agreed, changed them all to publish if ok